<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QWidget" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #CCE5FF; </string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" colspan="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="btn_remove">
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>55</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FF8000; border-radius: 5px; color: #FFF; font-size: 18px; border: 1px solid #FF0000;  margin: 0px 15px;</string>
     </property>
     <property name="text">
      <string>Remove Line Breaks</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">margin: 0 15px;</string>
     </property>
     <property name="text">
      <string>Paste your text in the box below and then click the button.

The new text will appear in the box at the bottom of the page.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">margin: 0 15px;</string>
     </property>
     <property name="text">
      <string>New Text without Line Breaks</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTextEdit" name="txt_input">
     <property name="styleSheet">
      <string notr="true">background-color: #FFF; border: 0px; border-radius: 5px; margin: 5 15px;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>56</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">margin: 0 15px;</string>
     </property>
     <property name="text">
      <string>Remove Line Breaks</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="btn_reset">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>55</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #D08E8E; border-radius: 5px; color: #FFF; font-size: 18px; border: 1px solid #FF0000;  margin: 0px 15px;</string>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QTextEdit" name="txt_result">
     <property name="styleSheet">
      <string notr="true">background-color: #FFF; border: 0px; border-radius: 5px; margin: 5 15px;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
